lib/api/controllers/funnelController.js

```diff
@@ -37,24 +37,25 @@ module.exports = function FunnelController (kuzzle) {
     var
       deferred = q.defer();
 
+    context.kuzzle = kuzzle;
```

Comment:
We cannot put kuzzle in the context. The context describes the current connection (cf HttpContext in [Java](https://docs.oracle.com/javase/7/docs/api/javax/xml/ws/spi/http/HttpContext.html) or [.net](https://msdn.microsoft.com/en-us/library/system.web.httpcontext(v=vs.110).aspx))

The context object is later one given to the role validation, and passed to the role validation methods that the bo users may have defined. We do not want an end-user to be able to modify kuzzle's behaviour on runtime.

As far as I can see, the only use is to check if the requested index & collection exist. If we only need the `kuzzle.indexes` collection, I would rather pass it to `role.prototype.isActionAllowed` method.